spec/models/administrateur_spec.rb

```diff
@@ -39,6 +32,13 @@
       it { is_expected.to be true }
     end
 
+    context "when the administrateur has a procedure where he/she is the only admin" do
```

Comment:
Si on veut dire que ça peut être un homme, une femme, ou whatever, on peut même utiliser `they` au singulier \o/ 

https://en.wikipedia.org/wiki/Singular_they

```suggestion
    context "when the administrateur has a procedure where they are the only admin" do
```

Reply:
Merci, tu me l'apprends :)